src/core/server/graph/tenant/mutators/Users.ts

```diff
@@ -118,8 +120,24 @@ export const Users = (ctx: TenantContext) => ({
     ),
   deactivateToken: async (input: GQLDeactivateTokenInput) =>
     deactivateToken(ctx.mongo, ctx.tenant, ctx.user!, input.id),
+  // can these call the same function with different arguments?
+
+  updateUsername: async (input: GQLUpdateUsernameInput) =>
+    updateOwnUsername(
+      ctx.mongo,
+      ctx.mailerQueue,
+      ctx.tenant,
+      ctx.user!,
+      input.username
+    ),
   updateUserUsername: async (input: GQLUpdateUserUsernameInput) =>
-    updateUsername(ctx.mongo, ctx.tenant, input.userID, input.username),
+    updateUserUsername(
+      ctx.mongo,
+      ctx.tenant,
+      input.userID,
+      input.username,
+      ctx.user!
+    ),
```

Comment:
oops this is probably broken bc the methods have the same name.

Reply:
er wait maybe it's not, but I had issues earlier with tslint being mad if the method signatures weren't identical if the names were the same?